crates/bevy_reflect/bevy_reflect_derive/src/lib.rs

```diff
@@ -60,11 +142,30 @@ pub fn derive_reflect(input: TokenStream) -> TokenStream {
 
 /// Derives the `FromReflect` trait.
 ///
-/// This macro supports the following field attributes:
-/// * `#[reflect(ignore)]`: Ignores the field. This requires the field to implement [`Default`].
-/// * `#[reflect(default)]`: If the field's value cannot be read, uses its [`Default`] implementation.
-/// * `#[reflect(default = "some_func")]`: If the field's value cannot be read, uses the function with the given name.
+/// # Field Attributes
+///
+/// ## `#[reflect(ignore)]`
 ///
+/// The `#[reflect(ignore)]` attribute is shared with the [`#[derive(Reflect)]`](Reflect) macro and has much of the same
+/// functionality in that it marks a field to be ignored by the reflection API.
```

Comment:
Nitpick: This reads better to me as "marks that a field will be ignored by the reflection API"